frontend/components/DateTime/DateTime.jsx

```diff
@@ -0,0 +1,220 @@
+'use strict'
+
+import {h, Component} from 'preact'
+import proptypes from 'proptypes'
+
+const fecha = require('fecha')
+
+const SECOND = 1000
+const MINUTE = 60 * SECOND
+const HOUR = 60 * MINUTE
+const DAY = 24 * HOUR
+const WEEK = 7 * DAY
+const YEAR = 365 * DAY
+const MONTH = YEAR / 12
+
+/**
+ * The time elapsed since a given event, as an automatically updated string.
+ */
+export default class ElapsedTime extends Component {
+  static propTypes = {
+    /**
```

Comment:
If you look at the other component (except perhaps a few that have been missed recently), we don't use that spacing. That is consistent with our linting rules that determine that comments don't need extra lines when at the beginning of a block or object literal (see [here](https://github.com/dadi/publish/blob/master/.eslintrc.json#L68-L70)).